src/detrotra.f90

```diff
@@ -0,0 +1,72 @@
+! This file is part of xtb.
+!
+! Copyright (C) 2017-2020 Stefan Grimme
+!
+! xtb is free software: you can redistribute it and/or modify it under
+! the terms of the GNU Lesser General Public License as published by
+! the Free Software Foundation, either version 3 of the License, or
+! (at your option) any later version.
+!
+! xtb is distributed in the hope that it will be useful,
+! but WITHOUT ANY WARRANTY; without even the implied warranty of
+! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+! GNU Lesser General Public License for more details.
+!
+! You should have received a copy of the GNU Lesser General Public License
+! along with xtb.  If not, see <https://www.gnu.org/licenses/>.
+
+subroutine detrotra4(linear,mol,h,eig)
```

Comment:
A module would be nice to always have an explicit interface.

Reply:
now using a module